adapters/osx-mdfind/index.js

```diff
@@ -0,0 +1,167 @@
+const fs = require('fs')
+const path = require('path')
+const app2png = require('app2png')
+const fuzzyfind = require('fuzzyfind')
+const { spawn } = require('child_process')
+let currentWorkingDirectory = null
+
+class File {
+  constructor (filePath, fileName) {
+    this.path = filePath
+    this.name = fileName
+    this.iconPath = path.join(
+      currentWorkingDirectory,
+      `./adapters/osx-mdfind/icons/${path.basename(this.path)}.png`
```

Comment:
It might be better to store the icons in the `./data/icons` directory, so the temp/cache related files can be centralized.